src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -159,7 +157,7 @@ public function createTargetGroup($name)
         return $targetGroup;
     }
 
-    private function createType($name)
+    private function createType($name): MediaType
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes